tests/mtouch/RegistrarTest.cs

```diff
@@ -334,6 +335,14 @@ public void MT4134 ()
 					new { Framework = "CarPlay", Version = "12.0" },
 					new { Framework = "IdentityLookupUI", Version = "12.0" },
 					new { Framework = "NaturalLanguage", Version = "12.0" },
+					new { Framework = "VisionKit", Version = "13.0" },
+					new { Framework = "SoundAnalysis", Version = "13.0" },
+					new { Framework = "PencilKit", Version = "13.0" },
+					new { Framework = "MetricKit", Version = "13.0" },
+					new { Framework = "LinkPresentation", Version = "13.0" },
+					new { Framework = "CoreHaptics", Version = "13.0" },
+					new { Framework = "BackgroundTasks", Version = "13.0" },
+					new { Framework = "QuickLookThumbnailing", Version = "13.0" },
```

Comment:
@dalexsoto We did not add any new framework, right? since in that case this test would need to be updated.

Reply:
Nope, no new frameworks yet